src/main/template/signature.handlebars

```diff
@@ -1,11 +1,25 @@
 <div role="tabpanel" class="guide">
     <h5>{{type}}</h5>
     <nav class="nav" role="tablist">
-        <a class="nav-link active" href="#sample-{{id}}" aria-controls="home" role="tab" data-toggle="tab">Sample</a>
-        {{#if signature}}<a class="nav-link active" href="#schema-{{id}}" aria-controls="tab" role="tab" data-toggle="tab">Schema</a>{{/if}}
+        <a class="nav-link active"
+            href="#sample-{{id}}"
+            aria-controls="home"
+            role="tab"
+            data-toggle="tab">
+            Sample
+        </a>
+        {{#if signature}}<a class="nav-link"
+            href="#schema-{{id}}"
+            aria-controls="tab"
+            role="tab"
+            data-toggle="tab">
```

Comment:
@hsingh23 about your comment: best I can tell, these controls have something to do with this commit: https://github.com/Cimpress-MCP/swagger-ui-cimpress/commit/ec1c90f33e72801212ada878f3ee5a11de417ef4